src/main/flight/mixer.c

```diff
@@ -785,7 +821,7 @@ FAST_CODE_NOINLINE void mixTable(timeUs_t currentTimeUs, uint8_t vbatPidCompensa
         throttle = applyThrottleLimit(throttle);
     }
 
-    const bool airmodeEnabled = airmodeIsEnabled();
+    const bool airmodeEnabled = airmodeIsEnabled() || launchControlActive;
```

Comment:
This should probably be renamed to indicate it's not only used for airmode.

Reply:
In this case the function hasn't changed. Basically the logic is forcing airmode on regardless of the feature or mode switch setting.  Besides that the behavior of `airmodeEnabled` is unchanged in the mixer.